components/script/dom/bluetooth.rs

```diff
@@ -60,6 +69,19 @@ impl Bluetooth {
                            BluetoothBinding::Wrap)
     }
 
+    pub fn get_service_map(&self) -> &DOMRefCell<HashMap<String, MutHeap<JS<BluetoothRemoteGATTService>>>> {
+        &self.service_instance_map
+    }
+
+    pub fn get_characteristic_map(&self)
+                                  -> &DOMRefCell<HashMap<String, MutHeap<JS<BluetoothRemoteGATTCharacteristic>>>> {
```

Comment:
I still suggest you move the `->` to one (or maybe two) tab size from the beginning.